### PR TITLE
Pin fsspec<2022.8

### PR DESCRIPTION
**Pull request by tgaddair**

### CODE REVIEW

**Review by arnavgarg1 — APPROVED** [review]
Nice!

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -96,7 +96,7 @@ jobs:
           if [ "$PYTORCH" == "nightly" ]; then
             cat requirements.txt | sed '/^torch[>=<]/d' | sed '/^torchtext[>=<]/d' | sed '/^torchvision[>=<]/d' | sed '/^torchaudio[>=<]/d' > requirements-temp && mv requirements-temp requirements.txt
             extra_index_url=https://download.pytorch.org/whl/nightly/cpu
-            pip install --pre torch torchtext torchvision torchaudio --extra-index-url $extra_index_url
+            pip install --pre torch==1.13.0.dev20220819+cpu torchtext torchvision torchaudio --extra-index-url $extra_index_url
```
Comment (arnavgarg1) [inline .github/workflows/pytest.yml:99]: Do we need to pin the torch version?

**Review by dantreiman — APPROVED** [review]
LGTM

**Comment by tgaddair** [comment]
Closing because bad fsspec version was yanked from pypi.

**Pull request closed without merging**